mods/seasonal/statuses.js

```diff
@@ -465,7 +465,7 @@ let BattleStatuses = {
 			this.add(`-anim`, target, 'Geomancy', target);
 			target.formeChange(forme);
 			target.setAbility('Anticipation');
```

Comment:
*cough*

Reply:
How did I miss this

Reply:
dont forget the hint too

Reply:
The hint is fine. why this isnt adaptability us beyond me